### ISSUE TITLE
Homebrew formula for OpenObserve

### ISSUE THREAD

**Issue by bbkane** [openobserve/openobserve#5029]
### Which OpenObserve functionalities are relevant/related to the feature request?

other

### Description

Installing OpenObserve is a pretty manual process - download the binary, set some environmental variables, create a systemd or launchd service, and start the service.

Though the docs really help with this process,  installing could be even easier!

### Proposed solution

Make a Homebrew formula for OpenObserve with some default settings (including, preferably a homebrew service) so folks could try it with one or two commands and automatically get updates when a new version comes out.

### Alternatives considered

This would be a local alternative to the free plan on the hosted service ( cloud.openobserve.ai ). I think this is an important alternative as it allows OpenObserve to be used without making an account or without internet service, which some people prefer.

**Comment by prabhatsharma** [comment]
Actually there is a faster way to install OpenObserve than brew

`curl -L https://raw.githubusercontent.com/openobserve/openobserve/main/download.sh | sh`

**Comment by bbkane** [comment]
Yes, but in addition to install, Homebrew also manages (and standardizes) other parts of the application lifecycle- updates, systems/launchd service creation and management, and uninstallation. 

```bash
brew install/upgrade/uninstall openobserve
brew services start/stop openobserve
```

Finally, installation via shell script is frowned upon by some (I don't personally think shell script installs are bad, but that's the world we live in...)
